.github/workflows/equivalence-test.yml

```diff
@@ -31,57 +27,89 @@ jobs:
     runs-on: ubuntu-latest
 
     outputs:
-      run-equivalence-tests: ${{ steps.metadata.outputs.run-equivalence-tests }}
-      terraform-version: ${{ steps.metadata.outputs.terraform-version }}
-      build-run-id: ${{ steps.metadata.outputs.build-run-id }}
-      target-branch: ${{ steps.metadata.outputs.target-branch }}
+      check-run-equivalence-tests: ${{ steps.check-run-equivalence-tests.outputs.check-run-equivalence-tests }}
+      terraform-version: ${{ steps.terraform-metadata.outputs.terraform-version }}
+      terraform-artifact-id: ${{ steps.terraform-metadata.outputs.terraform-artifact-id }}
+      target-branch: ${{ steps.target-branch.outputs.target-branch }}
 
     steps:
       - uses: actions/checkout@v3
         with:
           ref: ${{ github.event.workflow_run.head_branch }}
 
-      - id: metadata
+      - id: build-run-id
+        name: "Get build job ID"
         run: |
           if [[ "${{ github.event_name }}" == "workflow_dispatch" ]]; then
             # Then we map all our outputs from the user supplied inputs.
-            RUN_EQUIVALENCE_TESTS=true
-            TERRAFORM_VERSION=${{ inputs.terraform-version }}
             BUILD_RUN_ID=${{ inputs.build-run-id }}
           else
             # Quick sanity check, if the workflow_run that triggered this action
             # failed then we shouldn't carry on.
             if [[ "${{ github.event.workflow_run.conclusion }}" != "success" ]]; then
-              echo "::set-output name=run-equivalence-tests::false"
+              echo "::set-output name=check-run-equivalence-tests::false"
```

Comment:
This "set-output" mechanism has unfortunately been deprecated recently. Since we're changing and actively testing this right now anyway, I'd suggest updating to [the new approach](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/) at the same time even though it isn't strictly related to what this PR is about.

Based on some similar changes I've seen elsewhere I think it would look like this:

```bash
echo "check-run-equivalence-tests=false" >> $GITHUB_OUTPUT
```

(and of course similar for the other `::set-output` business elsewhere in this action. I won't comment on all of them because that would be distracting and noisy. :grinning: )

Reply:
Done!